qutebrowser/browser/browsertab.py

```diff
@@ -677,6 +677,7 @@ def _set_widget(self, widget):
 
         self._install_event_filter()
         self.zoom.set_default()
+        # pylint: enable=protected-access
```

Comment:
No need for this, as pylint already only turns things off for this function and it's needed for the entire function.